content/applications/general/calendars/outlook/outlook_calendar.rst

```diff
@@ -1,103 +1,138 @@
-   optional.
+To do add a client secret, click :guilabel:`Add a certificate or secret` and then click
+:guilabel:`New client secret`. Next, type a :guilabel:`Description` and select when the client
+secret :guilabel:`Expires`.
+
+Since resetting the synchronization can be tricky, Odoo recommends setting the maximum allowed
+expiration date for the client secret (24 months), so there is no need to re-synchronize soon.
+Finally, click :guilabel:`Add` to generate the client secret (:guilabel:`Secret ID`).
 
 Configuration in Odoo
 =====================
 
-Go to general *Settings* and activate the *Outlook Calendar* feature.
+In the Odoo database, go to :menuselection:`Settings --> General Settings --> Integrations`
+and activate the :guilabel:`Outlook Calendar` setting.
 
-.. image:: outlook_calendar/outlook_calendar.png
+.. image:: outlook_calendar/outlook-calendar-setting.png
    :align: center
-   :alt: Outlook Calendar feature activated in Odoo
+   :alt: The "Outlook Calendar" setting activated in Odoo.
 
-From your Azure portal, under *Overview*, copy your *Application (client) ID*, and paste it to
-*Client ID* in Odoo.
+From the Microsoft Azure portal, under the :guilabel:`Overview` section of the application, copy
+the :guilabel:`Application (Client) ID`, and paste it into the :guilabel:`Client ID` field in Odoo.
 
-.. image:: outlook_calendar/application_client_ID.png
+.. image:: outlook_calendar/client-id.png
    :align: center
-   :alt: Client ID token to be copied from Microsoft to Odoo
+   :alt: The "Client ID" in the Microsoft Azure portal.
 
-Going back to your Azure portal, now under *Certificates & secrets*, copy your *Client Secret Value*
-and paste it to *Client Secret* in Odoo.
+In the Microsoft Azure portal, under the :guilabel:`Certificates & secrets` section, copy the
+:guilabel:`Client Secret Value` and paste it into the :guilabel:`Client Secret` field in Odoo.
 
-.. image:: outlook_calendar/client_secrets.png
+.. image:: outlook_calendar/client-secret-value.png
    :align: center
-   :alt: Client Secret token to be copied from Microsoft to Odoo
+   :alt: The "Client Secret" token to be copied from Microsoft to Odoo.
+
+Finally, on the Odoo :menuselection:`Settings --> General Settings` page, click :guilabel:`Save`.
 
 .. _outlook_calendar/sync_with_outlook:
 
 Sync with Outlook
 =================
 
-.. important::
-   | Odoo Calendar sends a confirmation email at the creation of an event to its attendees. But, on
-     the **first** sync, Outlook will also send confirmation emails of all scheduled events
-     (including past ones).
-   | To avoid this situation, we recommend *archiving all past events, and removing yourself from
-     future ones in Odoo Calendar before the first sync*. This will prevent events from being
-     created in Outlook, and therefore, no redundant emails will be sent during the first
-     synchronization. After the sync, events will only send one confirmation email.
+Warnings (before sync)
```

Comment:
Optional: some/most/all of this section could be placed in a `.. warning::` block which could call further attention to it, given the yellow background.